src/components/forms/DocumentForm.tsx

```diff
@@ -149,9 +149,12 @@ export const DocumentForm = ({
             <FormLabel>Title</FormLabel>
             <Input bg="white" {...register('title')} />
           </FormControl>
-          <FormControl isRequired>
+          <FormControl isRequired isInvalid={errors.source_url ? true : false}>
```

Comment:
Could have the shorthand `isInvalid={errors.source_url}`

Reply:
Surely this is required even if there is no error? :shrug: